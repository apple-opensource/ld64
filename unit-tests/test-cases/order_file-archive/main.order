_foo2
_main
libfoo.a(foo.o):_foo3
_foo1

