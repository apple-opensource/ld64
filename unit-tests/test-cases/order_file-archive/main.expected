_foo2
_main
_foo3
_foo1
